docs/docsite/rst/contributing/creating_role.rst

```diff
@@ -0,0 +1,196 @@
+.. _creating_content_roles:
+
+**************
+Creating Roles
+**************
+
+.. contents:: Topics
+
+This topic describes how to create Ansible roles that can be imported into Galaxy.
+
+.. _creating_roles:
+
+Roles
+=====
+
+If you're unfamiliar with the concept of an Ansible role, view :ref:`ansible_roles`.
+
+Using the ``ansible-galaxy`` command line tool that comes bundled with Ansible, you can create a role with the ``init`` command.
+For example, the following will create a role directory structure called ``test-role-1`` in the current working directory:
+
+.. code-block:: bash
+
+    $ ansible-galaxy init test-role-1
+
+The *test-role-1* directory will contain the following:
+
+.. code-block:: bash
+
+    .travis.yml
+    README.md
+    defaults/
+        main.yml
+    files/
+    handlers/
+        main.yml
+    meta/
+        main.yml
+    tasks/
+        main.yml
+    templates/
+    tests/
+        inventory
+        test.yml
+    vars/
+        main.yml
+
+For a full explanation of all subdirectories and files listed above, and how they're used by Ansible, view the
+`Roles topic at the Ansible docs site <https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html?highlight=roles>`_
```

Comment:
1) Are you sure you want to highlight "roles" substring all over the page?
2) It's possible to use intersphinx to refer to the docs in the other sphinx site. I think it should be prefered over URL hardcoding